src/core/symbol/external/nacl_catapult.ts

```diff
@@ -0,0 +1,781 @@
+/*
```

Comment:
Question @Jaguar0625:
the tweetnacl package don't have the functions we used to derive shared secret exposed.  using the package's scalarmult(n,p) would produce different results?

Reply:
ok, we can skip this for now, but add an issue for it. afaik, the only reason we have special nacl file for NEM is because we're using a non-standard hash.